content/dev/code/forms.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Building a Form 
+description: Creating a new form for your shopping cart
+category: code
+weight: 10
+---
+
+Adding a custom form to your site is a common customization.  There are a 
+few guidelines for when you do this: 
+
+- Submit your form data using POST rather than GET. The Addon Conversion guide explains [the advantages of POST over GET](/dev/plugins/upgrading_to_1.5/#rewriting-addon-admin-pages-to-use-form-posts-instead-of-gets). 
```

Comment:
Perhaps this first point should begin as `Always submit` instead of `Submit`.

In fact, how about being even more emphatic:
```diff
- - Submit your form data using POST rather than GET. The Addon Conversion guide explains [the advantages of POST over GET](/dev/plugins/upgrading_to_1.5/#rewriting-addon-admin-pages-to-use-form-posts-instead-of-gets). 
+ - Always [submit your form data using POST](/dev/plugins/upgrading_to_1.5/#rewriting-addon-admin-pages-to-use-form-posts-instead-of-gets). Never use GET for forms. 
```